core/application/commands.go

```diff
@@ -37,6 +37,29 @@ func (a *app) CreateMicroVM(ctx context.Context, mvm *models.MicroVM) (*models.M
 		return nil, fmt.Errorf("an error occurred when attempting to validate microvm spec: %w", validErr)
 	}
 
+	if mvm.Spec.RootVolume.Source.VirtioFS != nil {
+		return nil, fmt.Errorf("VirtioFS is not available yet for RootVolume")
+	}
+	if len(mvm.Spec.AdditionalVolumes) > 0 {
```

Comment:
Its probably worth checking as well that if `volume.Source.VirtioFS` is specified that the microvm provider has the capability.

Reply:
I'll look into this one still work in progress